pkg/components/prometheus-operator/component.go

```diff
@@ -49,6 +49,7 @@ type component struct {
 	GrafanaAdminPassword string   `hcl:"grafana_admin_password,attr"`
 	Namespace            string   `hcl:"namespace,optional"`
 	EtcdEndpoints        []string `hcl:"etcd_endpoints,optional"`
+	EtcdPort             string   `hcl:"etcd_port,optional"`
```

Comment:
As etcd exposes several ports, this name is ambiguous. Is it the peer port, server port or metrics port? We also need a documentation for it (I know it's mentioned in the PR description, just making a anchor in the code for it).

Reply:
I assume it is under `prometheus operator` it will be a metrics port. Do you think we should be explicit about it?

Reply:
Ah right. I think it's fine then :+1: